### PR TITLE
Bettycrocker scraper

### PR DESCRIPTION
**Pull request by gregwa1953**
Added support for bettycrocker.com

### CODE REVIEW

**Comment by gregwa1953** [comment]
Please let me know if I did everything correctly, or if I missed any steps.

File: recipe_scrapers/tests/test_bettycrocker.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import os
+import unittest
+
+from recipe_scrapers.bettycrocker import BettyCrocker
+
+
+class TestAllRecipesScraper(unittest.TestCase):
```
Comment (hhursev) [inline recipe_scrapers/tests/test_bettycrocker.py]: Fantastic! Quick learner 😉 This here has to be `class TestBettyCrocker`. Everything else is perfect! 🍻 🍻

Comment (gregwa1953) [reply recipe_scrapers/tests/test_bettycrocker.py]: Sorry about.
I'll fix it now and re-submit.

Thank you.  

Just an FYI, I have one more ready to go through the PR process, probably today.  I have some other ideas that I'll work on implementing, probably one or two others within the week.

**Comment by hhursev** [comment]
Thanks!

**Pull request merged**